components/generic/CompareImageSlider.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <figure
+    ref="container"
+    class="compare-image"
+  >
+    <img
+      ref="imageLeft"
+      :src="imageLeft"
+      :alt="imageLeftText"
+      class="compare-image__image-left"
+      :style="leftImageClip"
+    >
+    <img
+      ref="imageRight"
+      :src="imageRight"
+      :alt="imageRightText"
+      class="compare-image__image-right"
+    >
+    <div
+      ref="slider"
+      class="compare-image__slider"
+      :style="sliderBarPosition"
+    >
+      <span class="compare-image__slider__bar" />
+      <button
+        :class="{ 'is-active' : dragging }"
+        class="compare-image__slider__handle"
+      >
+        <span class="sr-only">Slider Handle</span>
```

Comment:
Why is this span here?

Reply:
This is for accessibility. Using the class `sr-only` (screen reader only) hides the text from the browser but allows screen readers to let the user know what the component is.

Reply:
I am wondering if it makes sense here. When using a screen reader it is good to know that there is this image comparison component, but using a screen reader you cannot control the handler I guess. And most people that use a screen reader cannot actual see the comparison.

Reply:
Are we assuming all users who are blind are 100% blind?

Reply:
No, you are right. And not all screen readers users are not using a mouse. Leave it the way it is.